R/template.R

```diff
@@ -1,110 +1,184 @@
 #' Developer function for summarizing new metrics
 #'
-#' `metric_summarizer()` is useful alongside [metric_vec_template()] for
-#' implementing new custom metrics. `metric_summarizer()` calls the metric
-#' function inside `dplyr::summarise()`. `metric_vec_template()` is a
-#' generalized function that calls the core implementation of a metric function,
-#' and includes a number of checks on the types, lengths, and argument inputs.
+#' `numeric_metric_summarizer()`, `class_metric_summarizer()`, and
+#' `prob_metric_summarizer()` are useful alongside [metric_vec_template()] for
+#' implementing new custom metrics. These functions calls the metric function
+#' inside `dplyr::summarise()`. `metric_vec_template()` is a generalized
+#' function that calls the core implementation of a metric function, and
+#' includes a number of checks on the types, lengths, and argument inputs.
 #' See [Custom performance
 #' metrics](https://www.tidymodels.org/learn/develop/metrics/) for more
 #' information.
 #'
 #' @details
 #'
-#' `metric_summarizer()` is generally called from the data frame version
+#' `numeric_metric_summarizer()`, `class_metric_summarizer()`, and
+#' `prob_metric_summarizer()` are generally called from the data frame version
 #' of your metric function. It knows how to call your metric over grouped data
 #' frames and returns a `tibble` consistent with other metrics.
 #'
 #'
-#' @param metric_nm A single character representing the name of the metric to
+#' @param name A single character representing the name of the metric to
 #' use in the `tibble` output. This will be modified to include the type
 #' of averaging if appropriate.
 #'
-#' @param metric_fn The vector version of your custom metric function. It
+#' @param fn The vector version of your custom metric function. It
 #' generally takes `truth`, `estimate`, `na_rm`, and any other extra arguments
 #' needed to calculate the metric.
 #'
 #' @param data The data frame with `truth` and `estimate` columns passed
 #' in from the data frame version of your metric function that called
-#' `metric_summarizer()`.
+#' `numeric_metric_summarizer()`, `class_metric_summarizer()`, or
+#' `prob_metric_summarizer()`
 #'
 #' @param truth The unquoted column name corresponding to the `truth` column.
 #'
 #' @param estimate Generally, the unquoted column name corresponding to
 #' the `estimate` column. For metrics that take multiple columns through `...`
 #' like class probability metrics, this is a result of [dots_to_estimate()].
 #'
-#' @param estimator For numeric metrics, this is left as `NULL` so averaging
-#' is not passed on to the metric function implementation. For classification
-#' metrics, this can either be `NULL` for the default auto-selection of
-#' averaging (`"binary"` or `"macro"`), or a single character to pass along
-#' to the metric implementation describing the kind of averaging to use.
+#' @inheritParams rlang::args_dots_empty
```

Comment:
I typically put this above the first `@param` tag

Reply:
Done in https://github.com/tidymodels/yardstick/pull/322/commits/c5571802700963715ed46a4f32d59a3086dd6bcf